php-mode.el

```diff
@@ -1033,10 +1043,30 @@ PHP heredoc."
   "PHP Mode face used to highlight function names."
   :group 'php-faces)
 
+(defface php-function-caller-face '((t (:inherit default)))
+  "PHP Mode face used to highlight calling function names."
+  :group 'php-faces)
```

Comment:
Oh yes.  I got confused because the *caller* is usually either the object on which you call it or the class in which you use the object/method.

`php-function-call` is good and understandable, but does it work for both functions and methods, or only for methods?  If the latter, I would change it to `php-method-call`.